<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPlotUIClass</class>
 <widget class="QWidget" name="DataPlotUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iAssist - Plot Data</string>
  </property>
  <widget class="QwtPlot" name="plData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>981</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMeasurementProfileID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
